alex/applications/PublicTransportInfoCS/data/convert_idos_stops.py

```diff
@@ -28,7 +28,7 @@
 import sys
 import re
 from add_cities_to_stops import load_list, get_city_for_stop
-from alex.utils.various import remove_dups_stable
+from alex.utils.various import list_remove_duplicates_keep_ordering
```

Comment:
I'd rather keep the old name – I think it describes the function sufficiently and the new one makes lines too long ;-).